mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/JPAMailboxMapper.java

```diff
@@ -207,22 +198,6 @@ public void delete(Mailbox mailbox) throws MailboxException {
             .setParameter("namespaceParam", path.getNamespace())
             .setParameter("userParam", path.getUser());
     }
-
-    public void deleteAllMemberships() throws MailboxException {
-        try {
-            getEntityManager().createNamedQuery("deleteAllMemberships").executeUpdate();
-        } catch (PersistenceException e) {
-            throw new MailboxException("Delete of mailboxes failed", e);
-        } 
-    }
-    
-    public void deleteAllMailboxes() throws MailboxException {
-        try {
-            getEntityManager().createNamedQuery("deleteAllMailboxes").executeUpdate();
```

Comment:
you can also remove "deleteAllMailboxes" in mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/JPAMailbox.java